includes/settings.php

```diff
@@ -237,10 +237,12 @@ function license_key_callback() {
 		<input name="dt_settings[license_key]" type="text" placeholder="<?php esc_html_e( 'Registration Key', 'distributor' ); ?>" value="<?php echo esc_attr( $license_key ); ?>" id="dt_settings_license_key">
 	</div>
 
+	<?php if ( false === $settings['valid_license'] ) : ?>
```

Comment:
@dhanendran Nit picky comment but can you fix the indentation on this code block? Basically the paragraph within the `if` statement should be tabbed over one. Thanks!